src/js/utils.js

```diff
@@ -16,6 +16,27 @@ exports.is_sample_genetically_altered = function is_sample_genetically_altered(d
   || datum.protein !== undefined;
 };
 
+exports.makeD3SVGElement = function(tag) {
+  return d3.select(document.createElementNS('http://www.w3.org/2000/svg', tag));
+};
+
+exports.stableSort = function(arr, cmp) {
```

Comment:
That's right, `sortBy` takes a "metric function." In what way(s) are comparators more general? Why do you think underscorejs chose to do it this way if there was a more general option?